### PR TITLE
pubsub: convert iam to use gax

### PR DESCRIPTION
**Pull request by callmehiphop**
Relates to #2699

### CODE REVIEW

File: packages/pubsub/src/iam.js [hunk 1]
```diff
@@ -68,32 +63,16 @@ var util = require('util');
...
-
-  commonGrpc.Service.call(this, config, pubsub.options);
 }
 
-util.inherits(IAM, commonGrpc.Service);
-
 /**
  * Get the IAM policy
  *
+ * @param {object=} gaxOpts - Request configuration options, outlined
```
Comment (stephenplusplus) [inline packages/pubsub/src/iam.js]: Can you call these `gaxOptions`?

File: packages/pubsub/system-test/pubsub.js [hunk 2]
```diff
@@ -513,7 +513,6 @@ describe('pubsub', function() {
         assert.ifError(err);
 
         assert.deepEqual(policy.bindings, []);
-        assert.strictEqual(policy.etag, 'ACAB');
```
Comment (stephenplusplus) [inline packages/pubsub/system-test/pubsub.js]: What happens now?

Comment (callmehiphop) [reply packages/pubsub/system-test/pubsub.js]: It returns a Buffer, I tried decoding using the generated Policy object but that returns a ByteBuffer from protobufjs.

**Pull request merged**